.env.simple.auth.provider

```diff
@@ -0,0 +1,2 @@
+AUTH_SIMPLE_AUTH_ISSUER=localhost:3002
```

Comment:
Is this file needed given there is a .env.simple.auth.provider.default?

Reply:
Rename gone wrong - fixed it...